### DISCUSSION TITLE
Adobe Illustrator

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#20] [General]
# Adobe Illustrator

Check if Adobe Illustrator is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/d594e063-85b9-4c09-bcfc-980dc2b861d2

**Comment by talynone** [comment]
Officially supported in beta :
https://www.xda-developers.com/adobe-delivers-promise-illustrator-beta-windows-on-arm/

**Reply by OpenSource03** [reply]
Updated app to "Beta Support" Category
